tests/integration/olp-cpp-sdk-dataservice-read/CatalogClientCacheTest.cpp

```diff
@@ -60,6 +60,7 @@ class CatalogClientCacheTest : public CatalogClientTestBase {
     cache_ = std::make_shared<olp::cache::DefaultCache>(settings);
```

Comment:
check if you need cache_ as a member

Reply:
In tearDown we use ::Close() method from DefaultCache, which is not present in setting_.cache

Reply:
Yes, it is used in TeadDown, cannot be removed